app/models/configuration_script_payload.rb

```diff
@@ -6,4 +6,8 @@ class ConfigurationScriptPayload < ConfigurationScriptBase
   def self.base_model
     ConfigurationScriptPayload
   end
+
+  def run(*)
+    raise NotImplementedError, _("run must be implemented in a subclass")
+  end
```

Comment:
NOTE I want a consistent way to run ConfigurationScriptPayloads, EmbeddedAnsible has `run(vars)`, Workflow has `execute(userid:, input:)`, and Tower has `run_as_miq_job(options, userid = "system")` (this is in ConfigurationScript though, need to understand this vs ConfigurationScriptSource do we create the record first then run it?).

Hopefully eventually we can consolidate on `#run(vars, userid = "system")`